src/Failure.php

```diff
@@ -15,6 +18,14 @@ public function __construct(string $message, ?int $line = null)
         $this->line = $line;
     }
 
+    public static function fromLibXmlError(LibXMLError $error) : Failure
+    {
+        return new Failure(
```

Comment:
presence of `$message` and `$line` throws me off, but it's clear this is not a `Throwable`